### PR TITLE
Update more docs for UCX 1.11+

### PR DESCRIPTION
**Pull request by pentschev**

### CODE REVIEW

File: docs/source/examples/ucx.rst [hunk 1]
```diff
@@ -69,6 +72,9 @@ To start a cluster with all supported transports and an RMM pool:
     > --net-devices="auto" \
     > --rmm-pool-size="1GB"
 
+.. note::
+    For UCX 1.9 (deprecated) and older it's also necessary to set ``--net-devices="auto"``. UCX 1.11 and above is capable of selecting InfiniBand devices automatically.
```
Comment (charlesbluca) [inline docs/source/examples/ucx.rst:76]: Think for sake of consistency, we'll also want to remove `--net-devices="auto"` from the above code block

**Review by charlesbluca — COMMENTED** [review]
Thanks @pentschev! Just one small suggestion

File: docs/source/examples/ucx.rst (hunk 1 above)
Comment (pentschev) [reply docs/source/examples/ucx.rst:75]: Good catch, done in https://github.com/rapidsai/dask-cuda/pull/720/commits/e2b3e04cdf39f9405e221dbdb409a2dff41511bc .

**Review by charlesbluca — APPROVED** [review]

**Comment by pentschev** [comment]
Thanks @charlesbluca !

**Comment by pentschev** [comment]
@gpucibot merge

**Pull request merged**